src/custom/state/orders/updater.ts

```diff
@@ -312,7 +326,8 @@ export function ExpiredOrdersWatcher(): null {
         // validTo is either a Date or unix timestamp in seconds
         const validTo = typeof order.validTo === 'number' ? new Date(order.validTo * 1000) : order.validTo
 
-        return validTo < now
+        // let's get the current date, with our expired order validTo given a buffer time
+        return _validToWithBuffer(validTo, EXPIRED_ORDERS_BUFFER) < now
```

Comment:
I don't think we need to construct a new date. We can substract the 2 dates, and see if the difference if greater than the buffer

Reply:
:+1: